terraform/main.tf

```diff
@@ -112,6 +110,28 @@ module "pypi" {
   domain_map       = local.domain_map
 }
 
+module "test-pypi" {
+  source = "./warehouse"
+
+  name    = "Test PyPI"
+  zone_id = module.dns.primary_zone_id
+  domain  = "test.pypi.org"
+  extra_domains   = ["testpypi.python.org", "test.pypi.io", "warehouse-staging.python.org"]
```

Comment:
And come to think of it, the comment is probably best to be added near https://github.com/python/pypi-infra/pull/99/files#diff-6e34fc2014b1b31283651d165451c4f07ad68c90c3fc4c367e00cbbd95f983c5R4 too since it's kinda bizarre behavior.